lib/engines/hcktest/tests.rb

```diff
@@ -192,23 +198,36 @@ def status_count(status)
       @tests.count { |test| test['status'] == status }
     end
 
+    def results_count(status)
+      @tests_results.count { |test| test['status'] == status }
```

Comment:
The block parameter is named `test`, but it's a test result. `_1` will do anyway.